### PR TITLE
Add contact details at the bottom of the page

### PR DESCRIPTION
**Pull request by FinnERLewis**
Just testing / demo of pull requests.

### CODE REVIEW

File: docs/src/accessibility/README.md [hunk 1]
```diff
@@ -18,3 +18,8 @@ Accessibility is the responsibility of all contributors:
 - All development should be carried out with accessibility in mind.
 - An accessibility review should be carried out for any release featuring new or amended functionality.
 - If you need help please raise an issue on Github in the relevant issue queue.
+
+## Contat us for help
```
Comment (finnlewis) [inline docs/src/accessibility/README.md:22]: I think this is a typo

File: docs/src/accessibility/README.md [hunk 2]
```diff
@@ -18,3 +18,8 @@ Accessibility is the responsibility of all contributors:
...
+Feel free to contact us on Twitter: @localgovdrupal
+
```
Comment (finnlewis) [inline docs/src/accessibility/README.md:24]: Is there an email address?

**Review by finnlewis — CHANGES_REQUESTED** [review]

File: docs/src/accessibility/README.md (hunk 2 above)
Comment (richardaclarke) [reply docs/src/accessibility/README.md]: I think you should make the twitter profile into a hyperlink to https://twitter.com/LocalGovDrupal

**Review by finnlewis — APPROVED** [review]
Loogs good to me!

**Pull request merged**